src/UserAdmin/src/Smart.FA.Catalog.Infrastructure/Helpers/MinIoLinkGenerator.cs

```diff
@@ -19,9 +19,8 @@ public MinIoLinkGenerator(IOptions<S3StorageOptions> storageOptions)
     /// <inheritdoc />
     public string GenerateTrainerProfilePictureUrl(int trainerId, string imageFormat)
     {
-        // No unique salt because we should be able to decode a string
-        // TODO: rewrite upload system to allow impossible to decode url
-        var hashIds = new Hashids(MinioLinkDefaultSaltValues.TrainerProfilePicture, 8);
+        // Unique salt to prevent decoding from potential nefarious users
+        var hashIds = new Hashids(Guid.NewGuid().ToString(), 16);
```

Comment:
```suggestion
        var hashIds = new Hashids(salt: Guid.NewGuid().ToString(), 16);
```
Not storing the Guild.NewGuid in a '"var salt" variable is not intuitive therefore adding the argument will give more insight.